src/pages/[platform]/build-a-backend/functions/configure-functions/index.mdx

```diff
@@ -69,6 +69,17 @@ export const myDemoFunction = defineFunction({
 });
 ```
 
+## `ephemeralStorageSizeMB`
+
+By default, functions have 512MB of ephemeral storage to them. This can be configured from 512 MB upto 10240 MB. Note that this can increase the cost of function invocation. For more pricing information see [here](https://aws.amazon.com/lambda/pricing/).
```

Comment:
```suggestion
By default, functions have 512MB of ephemeral storage to them. This can be configured from 512 MB upto 10240 MB. Note that this can increase the cost of function invocation. For more pricing information visit the [Lambda pricing documentation](https://aws.amazon.com/lambda/pricing/).
```